website/docs/style_guide/about_style_guide.md

```diff
@@ -0,0 +1,65 @@
+# Robot Framework Style Guide
+
+## Purpose
+
+This style guide is a community driven set of sensible rules to write your Robot Framework code.
+
+As a starting point the existing standards from [Robot Framework](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#getting-started) user guide, [Robocop](https://github.com/MarketSquare/robotframework-robocop) and [Robotidy](https://robotidy.readthedocs.io/en/stable/).
```

Comment:
This sentence lacks some words at the end, like:
"As a starting point, the existing standards from this, this and that **are used**".